### PR TITLE
Add firebase_pro_weekly_offerers et aggregated_weekly_cultural_partner_activity (à partitionner)

### PR DESCRIPTION
**Pull request by mathilde-vey**
# DA PR

## Describe your changes

Please include a summary of the changes:

This PR [adds/removes/fixes/replaces] the [feature/bug/etc]. 

Tag a reviewer if necessacy  @github/username 

## Jira ticket number and/or notion link

JIRA-ticket_number

### Type of change
- [ ] Fix (non-breaking change which corrects expected behavior)
- [ ] New fields (non-breaking change)
- [x] New table (non-breaking change)
- [ ] Concept change (potentially breaking change which modifies fields according to new or evolving business concepts) 
- [ ] Table deletion (potentially breaking change which adds functionality/ table)
      
### Checklist before requesting a review
- [x] I have performed a self-review of my code
- [x] Fields have been snake_cased
- [x] I have checked my modifications don't break downstream models
- [ ] If my changes concern incremental table, I have altered their schema to accomodate with field's creation/deletion
- [ ] I have made corresponding changes to the [tables documentation](https://www.notion.so/passcultureapp/Documentation-Tables-175a397a8e854ff4a55ae4f3620dbe3b)
- [ ] I have made corresponding changes to the [fields glossary](https://www.notion.so/passcultureapp/854a436a8f1541e1b6ec2a65f8bab600?v=798024ba90404b139e5a17407a3bc604)
- [ ] I have updated the dag in cases of dependencies
- [ ] My code passes CI/CD tests
- [ ] I will post on slack review channel and ensure to specify the duration of the review task: short (<10min), medium (<30min), long (>30min)

### CODE REVIEW

File: orchestration/dags/dependencies/firebase/sql/analytics/firebase_pro_weekly_offerers.sql [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+        user_offerer.offerer_id
+        ,COUNT(distinct partner.partner_id) as nb_partner
+        ,COUNT(distinct user_offerer.user_id) as nb_user
+        ,date(offerer_creation_date) AS offerer_creation_date
+        ,CASE WHEN offerer.dms_accepted_at IS NOT NULL THEN date(offerer.dms_accepted_at)
+        WHEN offerer.dms_accepted_at IS NOT NULL AND is_synchro_adage IS TRUE THEN date(offerer_creation_date)
+        ELSE NULL END AS adage_synchro_date
+        ,weeks.week AS active_week
+        ,DATE_DIFF(weeks.week,DATE_TRUNC(offerer_creation_date,WEEK(MONDAY)),WEEK) AS nb_weeks_since_offerer_created
+        ,DATE_DIFF(CURRENT_DATE, offerer_creation_date, WEEK) AS nb_weeks_since_today
```
Comment (MatthieuRipollPassCulture) [inline orchestration/dags/dependencies/firebase/sql/analytics/firebase_pro_weekly_offerers.sql:20]: Peut être renommer ce champ en un truc plus intelligible type offerer_seniority_weeks

File: orchestration/dags/dependencies/firebase/sql/analytics/firebase_pro_weekly_offerers.sql [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+        ,COUNT(distinct unique_session_id) as nb_session
+        ,AVG(visit_duration_seconds) as avg_visit_duration_seconds
+        ,SUM(visit_duration_seconds) as sum_visit_duration_seconds
+
+-- offer management (modification and creation after hub)
+        ,COUNT(distinct CASE WHEN nb_start_individual_offer_creation+nb_confirmed_individual_offer_creation+nb_start_collective_offer_template_creation+nb_confirmed_collective_offer_template_creation+nb_start_collective_offer_bookable_creation+nb_confirmed_collective_offer_bookable_creation+nb_start_individual_offer_details_edition+nb_confirmed_individual_offer_details_edition+nb_start_individual_offer_stock_edition+nb_confirmed_individual_offer_stock_edition>0 THEN unique_session_id ELSE NULL END) AS nb_session_manage_offer
```
Comment (MatthieuRipollPassCulture) [inline orchestration/dags/dependencies/firebase/sql/analytics/firebase_pro_weekly_offerers.sql:26]: Concept à ranger dans firebase_pro_visit ?

File: orchestration/dags/dependencies/firebase/sql/analytics/firebase_pro_weekly_offerers.sql [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+
+-- guichet management
+        ,COUNT(distinct CASE WHEN nb_view_ticket_page>0 THEN unique_session_id ELSE NULL END) AS nb_session_manage_ticket
+-- booking management (see pages about booking)
+        ,COUNT(distinct CASE WHEN nb_view_individual_booking_page+nb_view_collective_booking_page>0 THEN unique_session_id ELSE NULL END) AS nb_session_manage_booking
+-- finance management (consult finance pages and add bank account)
+        ,COUNT(distinct CASE WHEN nb_view_financial_receipt_page+nb_view_financial_details_page+nb_view_banking_info_page+nb_clic_add_bank_account>0 THEN unique_session_id ELSE NULL END) AS nb_session_manage_finance
```
Comment (MatthieuRipollPassCulture) [inline orchestration/dags/dependencies/firebase/sql/analytics/firebase_pro_weekly_offerers.sql:33]: Concept à ranger dans firebase_pro_visit ?

File: orchestration/dags/dependencies/import_analytics/sql/analytics/aggregated_weekly_cultural_partner_activity.sql [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+        ) AS week
+),
+
+user_active_weeks AS (
+    SELECT
+        cultural_partner.partner_id
+        ,cultural_partner.offerer_id
+        ,date(partner_creation_date) as partner_creation_date
+        ,weeks.week AS active_week
+        ,DATE_DIFF(CURRENT_DATE, partner_creation_date, WEEK(MONDAY)) AS nb_weeks_since_today
```
Comment (MatthieuRipollPassCulture) [inline orchestration/dags/dependencies/import_analytics/sql/analytics/aggregated_weekly_cultural_partner_activity.sql:16]: partner_seniority_weeks ?

File: orchestration/dags/dependencies/firebase/sql/analytics/firebase_pro_weekly_offerers.sql (hunk 1 above)
Comment (mathilde-vey) [reply orchestration/dags/dependencies/firebase/sql/analytics/firebase_pro_weekly_offerers.sql:20]: oui carrément!

File: orchestration/dags/dependencies/firebase/sql/analytics/firebase_pro_weekly_offerers.sql (hunk 2 above)
Comment (mathilde-vey) [reply orchestration/dags/dependencies/firebase/sql/analytics/firebase_pro_weekly_offerers.sql:26]: firebase_pro_visits est une table clean et micro dans la catégorisation des événements, je referai une autre table en analytics pour ajouter ces concepts par session!

**Pull request closed without merging**